spec/controllers/orders/create_spec.rb

```diff
@@ -0,0 +1,55 @@
+RSpec.describe OrdersController, type: :controller do
+        
+  def the_action(minutes)
+    # post :create, :order => ourney_id: journey.id, minutes:minutes
+    # post :create, order: { ourney_id: journey.id, minutes:minutes }
+    post :create, params: { journey_id: journey.id, minutes:minutes }
+  end
+
+  describe '#create' do
+    
+    let!(:reference){ FactoryGirl.create :order}
+
+    context "when signed in" do
+      
+      sign_as
+      
+      let!(:car){ FactoryGirl.create :car, user: user }
+      let!(:journey){ FactoryGirl.create :journey, car: car }
+      
+      context "for a new order is created" do
+          
+        it "works" do
+          # expect{the_action(reference.minutes)}.to change(Order, :count).by(1)
+        end
```

Comment:
This example is currently commented out, as the `the action` for this example is not working since the params is missing `order`

![screen shot 2017-04-12 at 09 08 29](https://cloud.githubusercontent.com/assets/16335156/24955801/d97b3b04-1f7c-11e7-838c-71bef17d5be6.png)

The correct params are the following:

![screen shot 2017-04-12 at 09 08 44](https://cloud.githubusercontent.com/assets/16335156/24955981/8f9a0a28-1f7d-11e7-833a-5de185b492fb.png)

If I remove `require(:order)` from the following (within `orders_controller.rb`)
```
def order_params
    params.require(:order).permit(:minutes).merge(journey_id: @journey.id, price_hour: @journey.car.price_hour)
end
```

it works, but it is not correct. How can I modify `the action`to include the order param?